go.mod

```diff
@@ -2,10 +2,13 @@ module github.com/crossplane/crossplane
 
 go 1.13
 
+replace github.com/crossplane/crossplane-runtime => github.com/hasheddan/crossplane-runtime v0.0.0-20200320144815-bb562fce2536
```

Comment:
Keeping in draft until merge of https://github.com/crossplane/crossplane-runtime/pull/133